app/controllers/assessments_controller.rb

```diff
@@ -450,6 +450,11 @@ def show
                     else
                       @cud
                     end
+    @attachments = if @cud.instructor?
+                     @course.attachments
```

Comment:
I think this should be `@assessment.attachments`, rather than `@course` so that the handouts shown are specific to the assessment page currently shown, rather than whole course